.golangci.yaml

```diff
@@ -156,11 +156,11 @@ issues:
       text: "SA1019: (.+).(GoVersion|LangVersion) is deprecated: use the global `run.go` instead."
     - path: pkg/golinters/unused.go
       text: "rangeValCopy: each iteration copies 160 bytes \\(consider pointers or indexing\\)"
-
-run:
-  timeout: 5m
```

Comment:
do you know how `timeout` was used?

Reply:
[Timeout](https://golangci-lint.run/usage/configuration/) for total work (default 1m0s)